updates/ryan_update_selektion_editor.sql

```diff
@@ -0,0 +1,29 @@
+SET FOREIGN_KEY_CHECKS = 0;
+
+-- Fremdschlüssel entfernen
+ALTER TABLE edition_hateditor DROP FOREIGN KEY edition_hateditor_EditorID;
+ALTER TABLE edition_hateditor DROP FOREIGN KEY edition_hateditor_EditionID;
+
+-- Datentypen ändern
+ALTER TABLE selektion_editor MODIFY ID int;
+ALTER TABLE edition_hateditor MODIFY EditorID INT NOT NULL;
```

Comment:
Da edition_hateditor auf eine Selektion zeigt sollte zusätzlich zu INT NOT NULL auch noch DEFAULT -1 gesetzt werden.